transifex/native/django/utils/gettext.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import unicode_literals
+
+import re
+
+VAR_FORMAT = 'variable_{cnt}'
+
+
+def source_string_to_native(string):
+    """Transform any gettext placeholders in the given string to Transifex
+    Native placeholders.
+
+    Usage:
+    >>> source_string_to_native('This is %s & %s')
+    <<< ('This is {variable_1} & {variable_2}', '['variable_1', 'variable_2']')
+
+    >>> source_string_to_native('This is %(foo)s and %(bar)s')
+    <<< ('This is {foo} and {bar}', ['foo', 'bar'])
+
+    :param unicode string: the source string
+    :return: a tuple, containing the new string (with Transifex Native syntax)
+        and a list of the names of all variables
+    :rtype: Tuple[unicode, list]
```

Comment:
This is Pythons `typing` syntax, and I was under the impression that some IDEs would recognize this for Python 2 and provide autocomplete etc. Apparently this isn't the case, so I'll revert to `tuple`.

Reply:
@dbendilas It is not just python, primitive type definition should be capitalized, e.g. `Int` instead of `int` but we went with the lower case conventions, (not a big deal at all)